.travis.yml

```diff
@@ -0,0 +1,24 @@
+
+dist: bionic
+sudo: false
```

Comment:
Do you need this? Our other travis files don't have it so I'm just curious what it's doing.

Reply:
Doesn't seem like it - I had pulled this from an example that had this which had issues with not having it set in Travis. It seems like that key has been deprecated now though so removed